server/src/models/userModel.ts

```diff
@@ -3,7 +3,7 @@ import { IUser } from "../interfaces/IUser";
 import Joi from "joi";
 import jwt from "jsonwebtoken";
 
-const JWT_SECRET = "dsfasefs$$WT#T#$T#$T$#^%GESG$%U*&^IVSDGRTG$E%";
+const JWT_SECRET = "dsfasefs$$WT#T#$T#$T$#^%GESG$%U*&^IVSDGRTG$E%"; //ayoo shouldnt this be in the env file??
```

Comment:
ya but ill change it when we use it? this is just from another project that i copied